code/modules/client/preference/loadout/loadout_general.dm

```diff
@@ -2,6 +2,20 @@
 	display_name = "a d20"
 	path = /obj/item/dice/d20
 
+/datum/gear/dicebag
+	display_name = "A bag of dice"
+	path = /obj/item/storage/pill_bottle/dice
+	cost = 2
+
+/datum/gear/coingold
+	display_name = "a gold coin"
+	path = /obj/item/coin/gold
```

Comment:
This will be abused by R&D mains. So, no go.

Reply:
Why specifically R&D mains? The coin doesn't go in the protolathe for materials no can it go in the deconstructer for levels. And I can't think of a way to break it down into a gold bar, I even tried putting it in the ORM.